src/ex3/cli/item_manager.js

```diff
@@ -0,0 +1,65 @@
+import PokemonClient from "./PokemonClient.js";
+import fs from "fs";
+import dotenv from "dotenv";
+dotenv.config();
+const envFile = process.env;
+const cleanFile = { data: [] };
+class ItemManager {
+	constructor() {
+		this.pokemonClient = new PokemonClient();
+	}
+
+	getAllItems() {
+		let file = fs.readFileSync(envFile.jsonFilePath);
+		return JSON.parse(file);
+	}
+
+	setAllItems(file) {
+		fs.writeFileSync(envFile.jsonFilePath, JSON.stringify(file), (err) => {
+			if (err) {
+				console.error(err);
+			}
+		});
+	}
+
+	addListItem(item) {
+		const file = this.getAllItems();
+		file.data = [...file.data, { value: item }];
+		this.setAllItems(file);
```

Comment:
Pay attention that if it the writeFileSync fails for some reason you continue as normal without throwing exception or checking (only console.log) but `fs.writeFileSync` errors can be very critical and can mutate your entire data source file and even making it unreadable or corrupted. 
For this task it is not needed but pay attention that error handling is a must in such cases ("in the real world")